### PR TITLE
memory game success card and app.jsx update

### PR DESCRIPTION
**Pull request by tmansuroglu**
#22

### CODE REVIEW

**Comment by tmansuroglu** [comment]
I think my locales folder is older for this one. I will take a look at it once all PRs are sent

File: src/App.jsx [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
-import PuzzlePage from "./containers/PuzzlePage/index";
 import { useTranslation } from "react-i18next";
+import PuzzlePage from "./containers/PuzzlePage/index";
```
Comment (louisrli) [inline src/App.jsx:6]: app.jsx shouldnt be changed in a PR for the success card

Comment (tmansuroglu) [reply src/App.jsx:6]: renamed PR

**Comment by tmansuroglu** [comment]
Locales folder fixed

File: package.json [hunk 2]
```diff
@@ -16,11 +16,11 @@
         "invariant": "^2.2.4",
         "react": "^16.14.0",
         "react-card-flip": "^1.0.11",
-        "react-dnd": "^11.1.3",
-        "react-dnd-html5-backend": "^11.1.3",
```
Comment (louisrli) [inline package.json:23]: why did these get deleted

**Review by louisrli — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (tmansuroglu) [reply package.json:23]: they are not deleted anymore

**Review by ssinikoglu — APPROVED** [review]

**Review by ibrahimaljarad — APPROVED** [review]

**Comment by tmansuroglu** [comment]
@Ammar-64  ready for merge

**Pull request merged**